visualize:
  mean:  False
visualize:
  sigma: False
visualize:
  pts:   True

visualize:
  resolution: 0.5

local_planning:
  g_zero: 100
  num_yaw_samples: 10

frame_id: "world_origin"

body:
  frame_id: "fcu"

camera:
  frame_id: "rgbd_front_pitched/aligned_depth_to_color"
  h_fov: 1.211259 # [rad]
  width: 1280
  height: 720
  min_distance: 0.3 # [m]
  max_distance: 3 # [m]
